homeassistant/components/sensor/gtfs.py

```diff
@@ -241,7 +241,7 @@ def update(self):
             self._departure = get_next_departure(
                 self._pygtfs, self.origin, self.destination, self._offset)
             if not self._departure:
-                self._state = 0
+                self._state = None
                 self._attributes = {'Info': 'No more departures today'}
                 if self._name == '':
                     self._name = (self._custom_name or DEFAULT_NAME)
```

Comment:
It's not good that we might change name and as a result entity_id between restarts depending on api return result.

Reply:
You are right, this can indeed create an unclear entity_id situation. The sensor also can have one of two default names, which is confusing. I'll open another PR to cover these naming issues. Thanks! 😃